docs/dev-environment-setup.md

```diff
@@ -35,11 +35,19 @@ Below are some useful extensions that we recommend using with Carbon:
 
 ### Node.js & NPM
 
-The recommended way to install `node` and `npm` is using [Node Version Manager (`nvm`)](https://github.com/nvm-sh/nvm).
-Once you have installed `nvm`, you should run `nvm install lts/hydrogen` which will install a suitable version of `node` and `npm`.
+The recommended way to install Node.js and npm is using [Node Version Manager (`nvm`)](https://github.com/nvm-sh/nvm).
```

Comment:
Question: as we're enforcing that anyone developing in Carbon uses node 20 in the `check_node_version` script maybe we should add something about that here?